user-guide/Troubleshooting/Procedures/Troubleshooting_SAML_Issues.md

```diff
@@ -0,0 +1,124 @@
+
+If you are accessing your DMA via the web, check if the user credential boxes are visible during the login process. If they are visible, this suggests the external authentication is not in use.
+
+If the external authentication is working properly, you will get the option to log in via identity provider.
+
+![Logging in via identity provider](~/user-guide/images/Logging_In_SAML.png)<br/>*DataMiner version 10.3.10*
+
+#### The password field is filled in
+
+If the password field is already filled in, leave it empty. Local logon attempts take precedence over external authentication.
+
+#### DataMiner fails to connect to the identity provider
+
+DataMiner may face difficulties connecting to the identity provider because of various reasons:
+
+- An incorrect ipMetadata URL in the *DataMiner.xml* file (when connecting through a URL)
+
+- Improperly formatted *ipMetadata.xml* file (when a local file is used)
+
+  Resolution:
+
+  - Ensure that the *ipMetadata.xml* file is correctly formatted, using tools like [Notepad++](https://notepad-plus-plus.org/downloads/) with an XML plugin. Missing `<?xml version="1.0" encoding="utf-8"?>` headers often lead to namespace issues.
+
+  - Be cautious with XML syntax restrictions, especially the use of "&," which should be replaced with `&amp;`.
+
+#### Using SAML together with a proxy
+
+When using SAML with a proxy, use the following configuration to avoid authentication issues:
+
+- For 32-bit processes, execute the following command: `netsh winhttp set proxy proxy-server="<ip>:<port>" bypass-list="localhost"`
+
+- In *SLnet.exe.config*, add the following input under `<configuration>.<System.net>`:
+
+  ```xml
+  <defaultProxy>
+    <proxy
+     proxyaddress=[proxy address]
+     bypassonlocal="true"
+   />
+  ```
+
+  Replace placeholders with the correct IP addresses, ports, and addresses.
+
+### Missing or incorrect attribute statements
+
+The SAML attribute names must always match those in the identity provider and the *DataMiner.xml* file.
+
+1. Go to the *C:\Skyline DataMiner* folder and open the *DataMiner.xml* file.
+
+1. Verify that the `<ExternalAuthentication>` tag is configured correctly and the tag name for your attribute matches the definition.
+
+   For example, if the claim name is "email," your *DataMiner.xml* emailClaim tag should be `<EmailClaim>email</EmailClaim>`.
+
+1. If issues persist, use the *SAML-tracer* tool, a browser extension that captures SAML traffic.
+
+   1. In your browser, add the SAML-tracer extension ([Chrome](https://chrome.google.com/webstore/detail/saml-tracer/mpdajninpobndbfcldcmbpnnbhibjmch), [Firefox](https://addons.mozilla.org/nl/firefox/addon/saml-tracer/)).
+
+   1. Go to `http(s)://[DMA name]/`.
+
+   1. Click the puzzle icon in the top-right corner of your browser and select *SAML-tracer* from the list of available extensions.
```

Comment:
Just to double-check: is this the same in each browser, or is this the way it is for a specific browser? If it's the latter, we should make this clear.

Reply:
I only ever use firefox so don't know for sure but usually browser plugins show up there, although in a recent firefox version some things get hidden by default and should be force-shown through plug-in settings

Chromium browser will act similar normally but there are always slight differences.

Reply:
I have tested this in both Firefox and Chrome. I was unable to find the SAML-tracer extension in Microsoft Edge. I believe this extension may only exist for these two types of browsers.

Firefox:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/19847474-19e6-45ba-b5d6-a0d2d992bf63)
Chrome:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/c0638d46-b47f-4190-9549-4eca88391c4f)

Reply:
In that case, could you change this part of the procedure so that it's clear from the start that you need to use either Firefox or Chrome for this?

Reply:
This part of the procedure was adjusted in commit [b09ba16](https://github.com/SkylineCommunications/dataminer-docs/pull/1980/commits/b09ba160067cf16c52c2f559cf9fc07eea54a520).